gulpfile.js

```diff
@@ -47,25 +38,25 @@ var argv = yargs
         .option('clear_datastore', {
           describe: 'Whether to clear all data storage'
         })
-        .option('use_minification', {
-          describe: 'Whether to build with minification'
-        })
         .option('prod_env', {
           describe: 'Whether to run server in prod mode'
         })
         .demand(['gae_devserver_path'])
         .argv;
     }).argv;
-var concat = require('gulp-concat');
+
 var gulp = require('gulp');
+var concat = require('gulp-concat');
+var exec = require('child_process').exec;
```

Comment:
Can we have `var childProcess` and then refer to childProcess.exec below? That would keep the import conventions consistent.

Reply:
Done.